### PR TITLE
code tested with compas v1.0

### PR DESCRIPTION
**Pull request by franaudo**
additional changes have been added to `changelog.md`

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -16,4 +16,5 @@ pytest >=3.2
 sphinx_compas_theme >=0.13
 sphinx >=3.4
 twine
+meshpy
```
Comment (brgcode) [inline requirements-dev.txt:19]: this is certainly not a dev requirement...

File: examples/introduction_script.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
 # Create empty Structure object
 
-mdl = Structure(name='introduction', path='/home/al/temp/')
+mdl = Structure(name='introduction', path='C:/temp/')
```
Comment (brgcode) [inline examples/introduction_script.py:18]: would be good if we could make this non-specific to windows

File: src/compas_fea/__init__.py [hunk 3]
```diff
@@ -30,7 +30,7 @@
 __copyright__ = 'Block Research Group'
 __license__ = 'MIT License'
 __email__ = ''
-__version__ = '0.3.0'
+__version__ = '0.4.0'
```
Comment (brgcode) [inline src/compas_fea/__init__.py:33]: are you locally already at `0.4.0`?
bumping the version should happen with `invoke` such that it is done throughout the package automatically...

File: src/compas_fea/cad/blender.py [hunk 4]
```diff
@@ -3,31 +3,29 @@
...
-except:
-    pass
-
-try:
-    import bpy
-except:
-    pass
+import compas
+if not compas.IPY:
+    try:
```
Comment (brgcode) [inline src/compas_fea/cad/blender.py:8]: this should just be `if compas.BLENDER` not `try-except`

File: src/compas_fea/utilities/meshing.py [hunk 5]
```diff
@@ -3,7 +3,7 @@
 from __future__ import division
 from __future__ import print_function
 
-from compas.geometry import add_vectors
+from compas.geometry import add_vectors, primitives
```
Comment (brgcode) [inline src/compas_fea/utilities/meshing.py:6]: would prefer that the primitives are imported explicitly

**Review by brgcode — APPROVED** [review]
the cad related try excepts should be handled by `compas_fea/cad/__init__.py` using the `BLENDER` and `RHINO` flags of `compas` but i can fix that after the merge...

File: examples/introduction_script.py (hunk 2 above)
Comment (franaudo) [reply examples/introduction_script.py:18]: not sure how though...this is the folder where all the analysis stuff happens. If I put `compas_fea.TEMP` then a bunch of crap is created in the rhino location (or gives me an error)

**Pull request merged**